arbor/include/arbor/cv_policy.hpp

```diff
@@ -182,7 +182,7 @@ struct cv_policy_every_segment: cv_policy_base {
 };
 
 inline cv_policy default_cv_policy() {
-    return cv_policy_fixed_per_branch(1);
+    return cv_policy_every_segment();
```

Comment:
This is a big change to the default behaviour, and will have us diverge from NEURON's default. Is there some discussion supporting this?

Reply:
I'm probably wrong, so I'll undo this. Yesterday @KirHarms discovered some deviation with Neuron, and we though this change would match Neuron better. Meanwhile understanding of the difference in definitions between Neuron and Arbor has improved, and new results support that.